.circleci/config.yml

```diff
@@ -74,6 +74,12 @@ mysql-container: &mysql-container
     MYSQL_ROOT_PASSWORD: ''
     MYSQL_DATABASE: circleci
 
+postgres-container: &postgres-container
+  image: circleci/postgres:10.5-alpine
```

Comment:
We should check with @andrewdavidmackenzie 
But I remember we should also support crunchy DB and as I see it is only 9.6

Reply:
CrunchyDB 3.3.0 is Postgres 10.5.